snap/snapcraft.yaml

```
@@ -15,6 +15,13 @@ architectures:
 parts:
   heroku:
     plugin: nil
+    override-pull: |
+      snapcraftctl pull
+      COMMIT_MSG=$(git log --pretty=oneline --abbrev-commit | head -n 1 | awk '{print $2}')
```

Another way to do this might be to leverage some of the built in stuff git log has:

* Instead of `| head -n 1`, we could use `git log --max-count 1` (or `-n` alias)  to pull the latest commit message.
* Instead of `| awk '{print $2}'`, we could use Git's [pretty format](https://git-scm.com/docs/pretty-formats) to print only the subject: `git log --pretty="format:%s"`
Putting that altogether looks this:

```suggestion
      COMMIT_MSG=$(git log --max-count 1 --pretty="format:%s")
```



Putting that all together, it looks like:

```